README.md

```diff
@@ -16,6 +16,21 @@ modules: {
 }
 ```
 
+You can specify properties to omit by setting an `exportOmitProperties` option. This option takes an array of field names. For example, if you wanted to exclude the `trash` field from the export, you'd configure your piece like this:
+
+```javascript
+modules: {
+  'apostrophe-pieces-export': {},
+  // other modules...
+  'my-module-that-extends-pieces': {
+    // Without this, there is no export functionality for this type.
+    // Not all types are great candidates for exports.
+    export: true,
+    exportOmitProperties: ['trash']
```

Comment:
`export` should become an object when you want to say "true, but with some custom configuration." This is consistent with how other similar modules play it.

```
export: {
  omitFields: [ 'trash' ]
}
```

Also, it should be `omitFields` to match what we're used to when talking about schema fields, that's the word we most often use. Non-schema properties wouldn't need to get "removed" because the only way they'd even be there is if you added them with custom logic.